server/modules/user/repository/userRepository.js

```diff
@@ -75,23 +75,45 @@ export const countUsersBySurveyId = async (surveyId, countSystemAdmins = false,
     (row) => Number(row.count)
   )
 
-export const fetchUsers = async ({ offset = 0, limit = null }, client = db) =>
-  client.map(
-    `
-    SELECT ${selectFieldsCommaSep}
+const _usersSelectQuery = ({ selectFields, sortBy = 'email', sortOrder = 'ASC' }) => {
+  // check sort by parameters
+  const orderByFieldBySortBy = {
+    email: 'email',
+    name: 'name',
+    last_login_time: 'status, last_login_time', // 'status' is used to group users that have accepted invitation, otherwise they have never logged in
```

Comment:
should we use the keys, i think that are repeated between files

Reply:
uhm, these are not the keys but the column names... and in some cases they are the same. Should we introduce an object with the column names?

Reply:
yep Let's keep at it is!